src/document/generator.ml

```diff
@@ -81,8 +81,23 @@ let attach_expansion ?(status = `Default) (eq, o, e) page text =
       DocumentedSrc.
         [ Alternative (Expansion { summary; url; status; expansion }) ]
 
-let doc_of_expansion ~decl_doc ~expansion_doc =
-  Comment.standalone decl_doc @ Comment.standalone expansion_doc
+let make_page title kind url ?(header_title = make_name_from_path url) comments
+    items =
```

Comment:
Could you use labels for everything here ?

Also, comments are already lists, no need to do take a list of lists.

There starts to be a lot of those helpers floating around, I think we'll need to put them in a dedicated module at some point.

Reply:
I don't think labels would be good here, arguments have all different types and some are expected to be literals, for example:
```
              make_page modname `Mod url [ t.doc; expansion_doc ] items
```

> Also, comments are already lists, no need to have a list of lists.

I wanted to abstract the code that knows what comments are and that concatenate them in a specific way, instead of repeating that everywhere.

I've pushed some refactoring making my intent clearer.